src/SIL.Machine.AspNetCore/Services/BuildProgress.cs

```diff
@@ -6,6 +6,10 @@ public class BuildProgress : IProgress<ProgressStatus>
     private readonly string _buildId;
     private ProgressStatus _prevStatus;
 
+    private DateTime lastReportTime = DateTime.Now;
```

Comment:
Private fields should be prefixed with an underscore.

Reply:
Done.